apps/AppStandalone/README.md

```diff
@@ -5,39 +5,156 @@ A new Flutter project.
 ## Getting Started
 
 FlutterFlow projects are built to run on the Flutter _stable_ release.
+
+---
+
+This guide walks you through the steps required to run the app locally on your machine. Please
+follow the instructions carefully to ensure a smooth setup.
+
+### Prerequisites
+
+Before starting, make sure you have the following installed:
+
+- Flutter SDK
+- Dart SDK
+- Xcode (for iOS)
+- Android Studio (for Android)
+- CocoaPods (for iOS dependencies)
+- Firebase account
+
+### Setup Instructions
+
+1. **Get Flutter Dependencies**:
+    ```
+    flutter pub get
+    ```
+
+2. **Install iOS Pods**:
+   Navigate to the iOS directory and install the CocoaPods dependencies:
+    ```
+    cd ios
+    pod install
+    pod repo update
+    ```
+
+3. **Environment Configuration**:
+   Rename the environment configuration file:
+    ```
+    mv .env.template .env
+    ```
+
+4. **Firebase Project**:
+    - Create a new project in Firebase Console.
+    - Activate the FlutterFire CLI tool:
+        ```
+        dart pub global activate flutterfire_cli
+        ```
+    - Configure FlutterFire with your Firebase project:
+        ```
+        flutterfire configure
+        ```
+      Follow the prompts to select your project and target platforms (Android, iOS, Web).
+
+5. **API Keys**:
+   Add your Firebase and other necessary API keys to the `.env` file.
+
+6. **Run Build Runner**:
+   Generate necessary files with Build Runner:
+    ```
+    dart run build_runner build
```

Comment:
This fails for me because my env file is missing the [supabase URL and key](https://github.com/BasedHardware/Friend/blob/main/apps/AppStandalone/lib/env/env.dart#L31-L34). The `.env` template currently shows only a [pinecone api key](https://github.com/BasedHardware/Friend/blob/main/apps/AppStandalone/.env.template#L2). Are we using pinecone or supabase for the standalone app?

Reply:
Okay per https://github.com/BasedHardware/Friend/pull/58 and https://discordapp.com/channels/1192313062041067520/1226079093859287051/1231048604852949052 it appears that supabase isn't a part of the standalone app anymore. I removed those env vars from the config and it worked again. PR for the change is in https://github.com/BasedHardware/Friend/pull/81

Reply:
Follow up in https://github.com/BasedHardware/Friend/pull/82 to remove supabase entirely from the standalone app.